tests/test_pre_commit_1_sync.py

```diff
@@ -93,3 +95,11 @@ def test_pre_commit_hook_sync(
 
     nb = read("test.ipynb")
     assert len(nb.cells) == 3
+
+    # finally we move the paired notebook to a subfolder
+    tmpdir.mkdir("subfolder")
+    os.rename("test.py", "subfolder/test.py")
```

Comment:
NIT: shutil.move should be used instead of os.rename . This can fail if you keep your git repos on a different drive/partition/filesystem from your tmpdir. Additionally, in cases like this where the filename remains the same, you can just specify the folder.

Reply:
Sure I'll do that, thanks @Skylion007